core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -298,7 +303,7 @@ class CityConstructions {
 
     fun addProductionPoints(productionToAdd: Int) {
         if (!inProgressConstructions.containsKey(currentConstructionFromQueue))
-            inProgressConstructions[currentConstructionFromQueue] = 0
+            inProgressConstructions[currentConstructionFromQueue] = 0 // Should this be `productionToAdd`?
```

Comment:
No, because we add that in the last line. I'd rather have 1 line for that, rather than if/else